src/coreclr/jit/layout.cpp

```diff
@@ -536,6 +536,12 @@ ClassLayout* ClassLayout::Create(Compiler* compiler, const ClassLayoutBuilder& b
     return newLayout;
 }
 
+//------------------------------------------------------------------------
+// HasGCByRef: //   Check if this classlayout has a TYP_BYREF GC pointer in it.
+//
+// Return value:
```

Comment:
Probably should use `ClassLayout::IsStackOnly` instead?

Reply:
Although this is probably not a PR for considering such changes.

Reply:
Yeah, once we implement the zeroing behavior this should be switched to check for whether the layout is a byref struct since that's what we settled on.